Comanche055/JET_SELECTION_LOGIC.agc

```diff
@@ -470,7 +470,7 @@ ACYJETS		OCT	34360
 # THAN A MINIMUM IMPULSE (14MS), ALL JET CHANNEL COMMANDS WILL BE HELD FIXED FROM THE START OF THE T5 PROGRAM FOR
 # AT LEAST 14MS UNTIL THE INITIALIZATION OF NEW COMMANDS.  MOREOVER, A 14MS ON-TIME WILL BE ADDED TO ANY ROTATIONAL
 # COMMANDS GENERATED BY THE MANUAL CONTROLS OR THE JET SWITCHING LOGIC, AND ALL TRANSLATION COMMANDS WILL BE
-# ACTIVE FOR AT LEAST ONE CYCLE OF THE T5 PROGRAM (.1SEC)
+# ACTIVE FOR ATLEAST ONE CYCLE OF THE T5 PROGRAM (.1SEC)
```

Comment:
It is `ATLEAST` on the scanned source of [Comanche 055 page 1050 LR0385](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1050.jpg).